### ISSUE TITLE
JS is single threaded

### ISSUE THREAD

**Issue by favna** [Hackzzila/krypton#6]
Say what you will but you cannot make a multithreaded anything in NodeJS. The V8 engine is inherently single threaded and there is nothing you can do about that.

This repo is using false advertising to lure in users.

Source: Literally every single page that explains how JavaScript and the V8 engine work. Go to NodeJS, watch a [presentation on the JS event loop](https://www.youtube.com/watch?v=cCOL7MC4Pl0) or read the V8 engine developer documentation, I don't care. They'll all tell you the same.

**Comment by kyranet** [comment]
Oh dear.

This is not JS, this is C++, and the -pthread flag is being [used](https://github.com/Hackzzila/krypton/blob/bdf5becac708c2d499ecd7d6f80edcfd1800f393/binding.gyp#L9).

To dumb it down, the C++ layer (can) run multithreaded, independent of the JS layer, which (always) runs singlethreaded.

And C++ built-ins not necessarily need to use V8 for its entirety, just the exposed code (for example V8's JSFunctions to be called from the JS side for built-in usage...), there are just no limits.

And also, this library is currently unmaintained and does not longer work.

Edit: Going further, interop between a single-threaded language (JS) and a multi-threaded one (C++, and [nowadays Wasm](https://developers.google.com/web/updates/2018/10/wasm-threads) also joined the battle) has been possible for a long time.
